src/engine/board.ts

```diff
@@ -2,13 +2,14 @@ import Player from './player';
 import GameSettings from './gameSettings';
 import Square from './square';
 import Piece from './pieces/piece';
+import King from "./pieces/king";
```

Comment:
General: Folosim varianta cu ; la final. In proiect va forta si linterul asta drept o eroare.

Reply:
ok, asa o sa fac de acum incolo